app/presenters/queries/expanded_link_set.rb

```diff
@@ -0,0 +1,41 @@
+module Presenters
+  module Queries
+    class ExpandedLinkSet
+      def initialize(link_set:, schema: ::Queries::LinkSetSchema::new.call)
+        @link_set = link_set
+        @schema = schema
+      end
+
+      def links
+        link_types.each_with_object({}) do |type, hash|
+          hash[type] ||= {}
+          expanded_links(type).map do |field|
+            hash[type][field] ||= {}
+            hash[type][field] =
+              web_content_item.send(field)
```

Comment:
I'm a bit weary of `send` here. Is there a non-send way of doing this? If not, could we use `public_send` instead.